pageserver/src/tenant.rs

```diff
@@ -1707,11 +1736,30 @@ impl Tenant {
                 }
             }
         });
+
+        if stopping {
+            let timelines_accessor = self.timelines.lock().unwrap();
+            let not_broken_timelines = timelines_accessor
+                .values()
+                .filter(|timeline| timeline.current_state() != TimelineState::Broken);
+            for timeline in not_broken_timelines {
+                timeline.set_state(TimelineState::Stopping);
+            }
+        }
```

Comment:
This reminded me of "don't ask, tell": we could just request all timelines to go stopping, and the broken ones could just ignore it?

Reply:
They do ignore it, see `Timeline::set_state` impl.
I think we're checking for `!TimelineState::Broken` so that we don't cause log noise?

IDK, it has been there before, I'm just moving the code.

Reply:
yeah I guess it's fine. resolving.